lib/Sema/CSSimplify.cpp

```diff
@@ -14999,6 +15105,7 @@ ConstraintSystem::SolutionKind ConstraintSystem::simplifyFixConstraint(
   case FixKind::AllowAssociatedValueMismatch:
   case FixKind::GenericArgumentsMismatch:
   case FixKind::AllowConcreteTypeSpecialization:
+  case FixKind::IgnoreThrownErrorMismatch:
```

Comment:
I wonder how impactful this fix is vs. i.e. `@escaping` mismatch should it be more like a conversion mismatch which is has impact of `2`?

Reply:
I don't have a good sense of what the right impact score should be. I'll go with your suggestion of `2`.

Reply:
If one overload choice has a mismatch on argument  or result type and another on `throws` should that be ambiguous?

Reply:
No, I would not think it should be ambiguous. Within a single module, we don't allow overloading on `throws` alone, but of course this overloading can happen across module boundaries.

Reply:
What I am trying to ask is what overload is going to be a better suggestion in such case or whether both overloads should be mentioned, that’s where the impact matters.